conditional/blueprints/member_management.py

```diff
@@ -35,6 +35,7 @@
 from conditional.util.ldap import ldap_set_housingpoints
 from conditional.util.ldap import ldap_get_active_members
 from conditional.util.ldap import ldap_get_member
+from conditional.util.ldap import ldap_get_current_students
 from conditional.util.ldap import _ldap_add_member_to_group as ldap_add_member_to_group
```

Comment:
Same as above with the private methods, although it's not technically part of this PR.